frontend/src/modules/scaffold/views/scaffoldListView.js

```diff
@@ -128,6 +128,12 @@ define([
   * Returns an apt string value from Modal.Item value
   */
   function getModalItemValueString(value) {
+    if (value === 0 || value === false) {
+      return value;
+    }
+    if (value === '') {
+      return '-';
```

Comment:
If we're going with the above, and assuming a non-value === undefined, we could just refactor [these lines](https://github.com/adaptlearning/adapt_authoring/blob/19ed29991650aa48fb6db81e086f6b74c9e1658a/frontend/src/modules/scaffold/views/scaffoldListView.js#L131-L139) into:
```javascript
if(value === undefined) {
  return '';
}
```
... rather than adding ifs for each case.

Reply:
An empty Array or Object would also be a falsy value. Not sure if `value` could be one of those tbh.

Happy to go with @taylortom suggesitons

Reply:
@lc-thomasberger, arrays and objects are truthys,

@taylortom, I'm worried that null would be printed with your logic. Happy to be proved wrong.

Reply:
Finally doing some practical testing with this and I'm not seeing any problematic falsys coming through (most likely due to the way we set defaults for fields). It'll probably be much cleaner to refactor to return early?
```js
if (typeof value !== 'object') {
  return value;
}
if(Array.isArray(value)) {
  return Origin.l10n.t('app.items', { smart_count: Object.keys(value).length });
}
// print nested name/value pairs
var pairs = '';
for (var name in value) {
  if(value.hasOwnProperty(name)) pairs += '<br />' + wrapSchemaTitle(name) + value[name];
}
return '<p class="list-item-modal-object">' + pairs + '</p>';
```